src/GovUk.Education.ExploreEducationStatistics.Content.Api/ViewModels/AllMethodologiesViewModels.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using GovUk.Education.ExploreEducationStatistics.Publisher.Model.ViewModels;
+
+namespace GovUk.Education.ExploreEducationStatistics.Content.Api.ViewModels
+{
+    public class AllMethodologiesThemeViewModel : IComparable<AllMethodologiesThemeViewModel>
+    {
+        public Guid Id { get; set; }
+        public string Title { get; set; }
+        public List<AllMethodologiesTopicViewModel> Topics { get; set; }
+
+        public void RemoveTopicLeafNodesAndSort()
+        {
```

Comment:
Cool, do we have any existing tests covering this node removal?  If not, it's probably an important one to add coverage to, as this is the place where we stop potentially-not-yet-public Publications from being included in the tree.

Reply:
Yes we do in the form of these three tests in MethodologyServiceTests

```
GetTree_ThemeWithoutTopicsIsNotIncluded
GetTree_ThemeWithoutPublicationsIsNotIncluded
GetTree_ThemeWithoutPublishedMethodologiesIsNotIncluded
```

but I will add some more direct coverage for these methods and update the PR. Thanks.

Reply:
More unit tests added.